attachments_for_content/attachments_for_content.php

```diff
@@ -37,6 +37,8 @@
  * @package	 Attachments
  * @since	 3.0
  */
+
+#[AllowDynamicProperties]
```

Comment:
I would prefer to avoid dynamic properties if I can help it as it makes bugs harder to track down. I will try to find where these properties are created and come back for more info.

Reply:
@parapente can delete #[AllowDynamicProperties].
After you create a new article from the front end and add an attachment file, I tried adding it twice, and you encountered an error with a deprecated message.

![image](https://github.com/user-attachments/assets/6b9c151a-7c89-4133-825e-e76940001824)

I hope this helps you with debuging the issue.